test/tests/functional/pbs_equiv_classes.py

```diff
@@ -1712,7 +1712,7 @@ def test_multiple_job_preemption_order(self):
                 deleted = True
                 break
             else:
-                time.sleep(1)
+                time.sleep(2)
```

Comment:
Please add a comment indicating why two seconds was chosen, in case someone tries to optimize wait times in the future.

Reply:
Done.